recipes/flang-activation/activate.sh

```diff
@@ -0,0 +1,17 @@
+export _OLD_FC=$FC
+export _OLD_CPATH=$CPATH
+export _OLD_FFLAGS=$FFLAGS
+export _OLD_LIBRARY_PATH=$LIBRARY_PATH
+export _OLD_FORTRANFLAGS=$FORTRANFLAGS
+
+if [ -z "$CONDA_PREFIX" ]; then
+    export CPATH=$CPATH:$PREFIX/include
+    export LIBRARY_PATH=$LIBRARY_PATH:$PREFIX/lib
```

Comment:
Let's not set these. We should just set the correct flags in `FFLAGS`

Reply:
Happy to! For context, is untouched from the existing [activation](https://github.com/conda-forge/flang-feedstock/blame/main/recipe/activate.sh) in flang.